client/landing/gutenboarding/onboarding-block/style.scss

```diff
@@ -6,34 +6,98 @@
 		display: none;
 	}
 }
+.onboarding-block__acquire-intent {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	margin-top: 40px;
+	position: fixed;
+
+	top: 20%;
+	left: 0%;
+	width: 100%;
+}
+
+.onboarding-block__questions {
+	display: inline-block;
+	min-width: 800px;
+}
 
 .onboarding-block__questions-heading {
-	font-size: 1.5em;
-	line-height: 2em;
 	margin-bottom: 10px;
+	text-align: left;
+	font-size: 20px;
+}
+
+.gutenboarding-block-list .components-form-token-field,
+.onboarding-block__question.selected {
+	font-size: 40px;
+	margin-top: 18px;
+	margin-bottom: 28px;
+
+	&:first-of-type {
+		margin-top: 0;
+	}
+
+	&:last-of-type {
+		margin-bottom: 0;
+	}
 }
 
 .gutenboarding-block-list .components-form-token-field,
 .onboarding-block__question {
 	display: flex;
-	font-size: 2em;
-	line-height: 2.1em;
+	font-size: 28px;
 	margin-bottom: 10px;
+	transition-property: 'font-size';
+	transition-duration: 0.25s;
+	transition-timing-function: 'linear';
+	transition-delay: 0;
 
-	> * {
-		margin: 0 0.5em 0 0;
-	}
 }
 
 .onboarding-block__multi-question {
 	list-style: none;
 	margin: 0;
 	padding: 0;
 
+	label {
+		margin-bottom: 36px;
+		display: inline-block;
+	}
+
+	label::after {
+		content: '✓';
+		margin-left: 8px;
+		opacity: 0;
+		font-size: 40px;
+		transition-property: opacity, color;
+		transition-duration: 0.3s;
+		transition-timing-function: 'linear';
+		transition-delay: 0;
+
+	}
+
+	li.selected label::after {
+		opacity: 1;
+		color: $light-gray-800;
+	}
+	
+	li.selected	label:hover::after,
+	label:hover::after {
```

Comment:
Whitespace is weird on this line and the line before :grimacing:

Reply:
Right! There's some tabs that shouldn't be there